### ISSUE TITLE
invisible bug

### ISSUE THREAD

**Issue by 53701337** [TwistedScorpio/Nostalrius#41]
I think this update https://github.com/TwistedScorpio/Nostalrius/commit/76c925111d468da7841fb2c74869b160b7fa77d8 make another bug
https://imgur.com/r9zBo4d

**Comment by nekiro** [comment]
yeah, you are right, no idea why the code for ghost mode was removed from there.
